src/routes/+page.svelte

```diff
@@ -1,2 +1,22 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<script>
+
+</script>
+
+<form action="#">
+    <div class="rows">
+    <div class="row">
+        <input class="button is-outlined" 
+            multiple 
+            type="file" 
+            name="media_uploads" 
+            id="media_uploads">
+   
+        <button class="button is-primary is-outlined">Send</button>
+    </div>
+</div>
+</form>
+
+<style>
+    @import "https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma-rtl.min.css";
```

Comment:
Please import bulma styles in one place. Also it should be global css, available for all files, like `src/app/app.css`. Or just link it in app.html.

Reply:
Done